virttest/qemu_monitor.py

```diff
@@ -2276,9 +2276,12 @@ def balloon(self, size):
         # Send a balloon monitor command
         self.send_args_cmd("%s value=%s" % (cmd, size))
         # Look for BALLOON QMP events
-        if not utils_misc.wait_for(lambda: self.get_event(qmp_event), 120):
-            raise QMPEventError(cmd, qmp_event, self.name)
-        logging.info("%s QMP event received" % qmp_event)
+        if self.verify_status('running'):
```

Comment:
verify_status()  always return None,  so code in if branch will not be execute. and is VM status is pause (in your case), this line will raise exception  virt_vm.VMStatusError.